web/src/muted_topics_ui.js

```diff
@@ -44,18 +45,28 @@ export function handle_topic_updates(user_topic) {
     rerender_for_muted_topic(old_muted_topics);
 }
 
-export function mute_topic(stream_id, topic, from_hotkey) {
-    const stream_name = stream_data.maybe_get_stream_name(stream_id);
+export function update_topic_visibility_policy(
+    stream_id,
+    topic,
+    new_visibility_policy,
+    from_hotkey,
+) {
     const data = {
         stream_id,
         topic,
-        op: "add",
+        visibility_policy: new_visibility_policy,
     };
 
-    channel.patch({
-        url: "/json/users/me/subscriptions/muted_topics",
+    channel.post({
+        url: "/json/user_topics",
         data,
         success() {
+            // Accidentally unmuting a topic isn't as much an issue as accidentally muting
```

Comment:
This comment should be updated, considering the fact that we have now other visibility policies.

Reply:
I will update this comment. Any suggestions for how to handle unmuted topic success?

Reply:
I think unmute topic success can be handled when working on hotkeys as the feedback_widget is only shown for the mute topic if it is done from hotkey.